examples/interaction-commands.hs

```diff
@@ -89,88 +103,117 @@ newExampleSlashCommand =
         d
           { createApplicationCommandOptions =
```

Comment:
Overall the tictactoe example is kinda bloated. It's not a blocker here, I'll merge it, but I might replace it with something simpler in the future

Reply:
I'll be honest, it did get away from me.